_posts/2018-09-29-analyzing-the-behaviour-of-python-function-splice.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Analyzing the behaviour of Python function slice"
+date: "2018-09-29 10:17:46 +0530"
+tag:
+  - python
+  - slice
+  - python trick questions
+---
+
+
+![Title Image](/assets/images/python_slice_function/title_image.jpg)
+
+Last Friday, I was sitting at one of the good coffee shop of Bangalore with my
```

Comment:
s/at/in/

I'd probably also say "one of my friends", which seems more natural to me.